package.json

```diff
@@ -23,10 +23,8 @@
     "type": "git",
     "url": "https://github.com/onefinestay/react-daterange-picker"
   },
-  "peerDependencies": {
-    "react": ">=0.12.0"
-  },
   "dependencies": {
+    "react": ">=0.12.0",
```

Comment:
I wasn't sure then as I am not sure now, but `npm` gives a funny warning when using `peerDependencies` and this seems to solve the issue. [Warning info here] (https://docs.npmjs.com/files/package.json#peerdependencies)